packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1123,11 +1009,6 @@ new PrismaClient({
           // before we send the execution request, we use the changed params
           const changedInternalParams = { ...internalParams, ...changedParams }
 
-          // TODO remove this once LRT is the default transaction mode
-          if (index > 0 && !this._hasPreviewFlag('interactiveTransactions')) {
-            delete changedInternalParams['transactionId']
-          }
```

Comment:
Do we need to undo this?

Reply:
No it is not needed anymore. Now both transactions can have a tx id that is either a `string` or a `number`:
* if it's a string, then it's a GUID for an interactive transaction
* if it's a number, then it's an id created by regular transactions

Depending on what kind of transaction it is, we tell different things to the QE:
* if it's an iTX, then the query is sent with a `transactionId` header to the QE
* if it's a rTX, then we send it as a batch to the QE with `runInTransaction = true`

So to answer the question, no I don't think this should be undone.
`delete changedInternalParams['transactionId']`

This could actually cause batching errors because tx would be id-less, see below:
* https://github.com/prisma/prisma/blob/b0af104d16accd2ed92b0ca1ff73847db1cce3b0/packages/client/src/runtime/RequestHandler.ts/#L81
* That means that non-awaited `$transaction` calls, could cause batches to be mixed up
  * It might be hard to test but I have left a todo for myself after this PR